tasks-by-users/Naufil-Shemle/01-e-commerce-website/index.html

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>RedStore | Ecommerce Website Design</title>
+    <link rel="stylesheet" href="style.css" />
+		<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+		
+		<link
+		rel="stylesheet"
+		href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.6.0/font/bootstrap-icons.css"
+		integrity="sha384-7ynz3n3tAGNUYFZD3cWe5PDcE36xj85vyFkawcF6tIwxvIecqKvfwLiaFdizhPpN"
+		crossorigin="anonymous"
+	/>
+
+	</head>
+  <body>
+
+		<div class="header">
+    <div class="container">
+      <div class="navbar">
+        <div class="logo">
+          <a href="index.html"><img src="images/logo.png" alt="Website Logo" ><a> 
+        </div>
+        <nav>
+          <ul id="MenuItems">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact</a></li>
+            <li><a href="account.html">Account</a></li>
+          </ul>
+        </nav>
+				<a href="cart.html"><img src="images/cart.png" alt="Cart Icon" ></a>
+				<img src="images/menu.png" alt="Menu icon" class="menu-icon" onclick="menutoggle()">
+      </div>
+      <div class="row">
+        <div class="col-2">
+          <h1>Give Your Workout <br> New Style</Style></h1>
+					<p>Success isn't always about greatness. It's about consistency. Consistent <br>hard work gains success. Greatness will come.</p>
+					<a href="" class="btn">Explore Now &#8594;</a>
+        </div>
+        <div class="col-2">
+					<img src="images/image1.png" alt="sports player image">
+				</div>
+      </div>
+    </div>
+	</div>
```

Comment:
Is your prettier working?? @Naufil-Shemle

Reply:
It's working in VS Code, I don't know why it's not working at here 🙄

Reply:
Check your VS code settings and format this please @Naufil-Shemle

Reply:
I discussed with @faran4engg, He said that leave this as it is.